instrumented-tests/http-server-mock/Sources/HTTPServerMock/ServerProcessRunner.swift

```diff
@@ -22,12 +22,13 @@ public class ServerProcessRunner {
     /// Waits until server is reachable.
     /// Returns `ServerProcess` instance if the server is running, `nil` otherwise.
     public func waitUntilServerIsReachable() -> ServerProcess? {
-        let deadline = Date(timeIntervalSinceNow: 5)
+        let deadline = Date(timeIntervalSinceNow: 10)
```

Comment:
Looks like the first server startup takes a bit longer on CI with py3..
I've increased the deadline and wait 0.5 seconds between pings.

Reply:
Let's give it a try 👍